### ISSUE TITLE
Need update to work with RN 0.47

### ISSUE THREAD

**Issue by Stevenfly** [zo0r/react-native-push-notification#533]
RN 0.47 compile error:

```
error: method does not override or implement a method from a supertype
    @Override
    ^
```

Is this package still being maintained right now? The last commit is in May 👎 :(

Also see #522 #500 #521 #532

**Comment by vbuch** [comment]
@Stevenfly this one has been merged so yours should be closable, right?

**Comment by rawrmaan** [comment]
Fixed in #532.
